app/Transformers/Api/Client/ServerTransformer.php

```diff
@@ -42,7 +42,10 @@ public function transform(Server $server): array
             'internal_id' => $server->id,
             'uuid' => $server->uuid,
             'name' => $server->name,
-            'node' => $server->node->name,
+            'node' => [
```

Comment:
I'm not sure if this is gonna fly.  This is a breaking API change and while I agree with the new schema, it may cause problems for users.  Maybe just a new field in the root named `is_node_under_maintenance` or something.  This should definitely be revisited in v2.